azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/di/DependencyInjectionContainerImpl.kt

```diff
@@ -272,7 +276,7 @@ internal class DependencyInjectionContainerImpl(
         )
     }
 
-    private val callingService by lazy {
+    override val callingService by lazy {
```

Comment:
Why start exposing now? Can we punch through in another way to get access to what you need?

I.e. when accessing getDebugInfo.getSupportFiles in CallingSDK, I pass only the necessary function through for mocking.

CallingService is a heavy dependency, so it's exposure should be limited if possible.